change_ttl/README.md

```diff
@@ -1,3 +1,56 @@
-# Tinybird Versions - {{ YOUR USE CASE NAME HERE }}
+# Tinybird Versions - Change a Data Source's TTL
+
+
+The Data Source's TTL determines how long data remains valid and accessible within a system. This process is crucial as it ensures data accuracy, relevance, and helps in managing system performance by refreshing or removing outdated information, thereby maintaining the integrity and efficiency of your data infrastructure.
+
+These are the steps in modifying TTL, ensuring there's no data loss during the process.
+
+## Step 1: Change the ENGINE_TTL setting in a Data Source
+
+Whether it's needed to add a TTL or to change an existing one, the setting to define the TTL is `ENGINE_TTL`
+
+1. Apply the changes to the file
+
+In this example, we're changing the TTL of an existing Data Source from 60 days to 10 days. First of all, we apply the change to the Data Source file `analytics_events.datasource`:
+
+```diff
+- ENGINE_TTL "timestamp + toIntervalDay(60)"
++ ENGINE_TTL "timestamp + toIntervalDay(10)"
+```
+
+2. Generate a new release
+
+```sh
+tb release generate --semver 0.0.1
+```
+
+3. Make sure the scripts use the `--yes` flag to force the deployment
+
+```sh
+#!/bin/bash
+set -e
+
+tb deploy --yes
+```
+
+4. Update the permissions
+
+Don't forget to give execution permissions to your scripts
```

Comment:
If I'm not wrong "tb release generate" command already gives the execution permissions to the deployment scripts.

Reply:
Yes, they're not enough for github/gitlab pipelines. I've commented that in an issue in our private repository. We could skip this step as this would be improved at some point.

Reply:
We have to review it so, thx!